ansible/files/postgresql_config/postgresql.conf.j2

```diff
@@ -763,6 +763,9 @@ jit_provider = 'llvmjit'		# JIT library to use
 # WAL-G specific configurations
 #include = '/etc/postgresql-custom/wal-g.conf'
 
+# read replica specific configurations
+#include = '/etc/postgresql-custom/read-replica.conf'
```

Comment:
Is there any reason to include it commented out here, only to uncomment it using sed/replace in all the artifacts we're producing? iirc we used this pattern for e.g. supautils config because we wanted to enable it only for some of the artifacts.

Reply:
yea - it can be left uncommented but I was just standardizing with the other custom config files we have above.